microsoft-edge/devtools-guide-chromium/speed/get-started.md

```diff
@@ -22,440 +22,480 @@ ms.date: 05/04/2021
+    ```css
+    .hero-image img {
+        width: 100%;
+        position: static;
+        aspect-ratio: 2.5;
+        object-fit: cover;
+    }
+    ```
 
-![The Overview section of the trace](../media/speed-glitch-tony-remix-performance-slow-network-slow-cpu-main-highlight.msft.png)
+    This time, the CSS rule already had a `width` and `object-fit` defined. But instead of defining a `max-height` for the image, you use the `aspect-ratio` property to make sure the browser knows exactly how much space to reserve for the image on desktop devices.
 
-Investigate the trace to find ways to do less JavaScript work:
+1. Save your changes in Visual Studio Code, then refresh the webpage in Microsoft Edge, and run a new audit in the **Lighthouse** tool to see how your changes affect the load performance:
 
-1. Click the **Timings** section to expand it.  Based on the fact that there may be a bunch of [Timing](https://developer.mozilla.org/docs/Web/API/User_Timing_API) measures from React, it seems like Tony's app is using the development mode of React.  Switching to the production mode of React may yield some easy performance wins.
+    Your score is now up to 37, but more importantly the **Cumulative Layout Shift** score is down to 0 indicating that there are no more shifts in the layout as the page loads.
 
-   ![The Timings section](../media/speed-glitch-tony-remix-performance-slow-network-slow-cpu-timings.msft.png)
+    ![The Lighthouse tool report showing a better score after the aspect-ratio changes](./images/lighthouse-report-with-aspect-ratio.png)
 
-1. Click **Timings** again to collapse that section.
+#### Do less main thread work
 
-1. Browse the **Main** section.  This section shows a chronological log of main thread activity, from left to right.  The y-axis (top to bottom) shows why events occurred.  For example, in the figyre after the following, the `Evaluate Script` event caused the `(anonymous)` function to run, which caused `(anonymous)` to run, which caused `__webpack__require__` to run, and so on.
+Your latest report shows high **Time to Interactive** and **Total Blocking Time** metric values, which means that there's still something on the webpage that takes a long time to run and prevents the page from being usable for several seconds.
 
-  ![The Main section](../media/speed-glitch-tony-remix-performance-slow-network-slow-cpu-main.msft.png)
+Scroll down to the **Diagnostics** section which says to **Minimize main-thread work** and **Reduce JavaScript execution time**. The main-thread is where the browser does most of the work that's needed to display a page, such as:
```

Comment:
```suggestion
Scroll down to the **Diagnostics** section which says to **Minimize main-thread work** and **Reduce JavaScript execution time**. The main thread is where the browser does most of the work that's needed to display a page, such as:
```
tech review this change.  if UI shows "main-thread", maybe ignore this suggested change